### PR TITLE
[Method References 2] LPS-171886 Convert method reference to ServiceTrackerMap in class MonitoringMessageListener(module portal-monitoring)

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by Kvale1733** [comment]
@W2jx No regressions found.

**Comment by W2jx** [comment]
no conflicts(12/30 14:57)

**Comment by W2jx** [comment]
no conflicts(1/3 9:07)

**Comment by W2jx** [comment]
no conflicts(1/4 9:03)

**Comment by W2jx** [comment]
no conflicts(1/5 8:59)

**Review by dantewang — APPROVED** [review]

File: modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/messaging/MonitoringMessageListener.java [hunk 1]
```diff
@@ -192,6 +185,12 @@ protected synchronized void unregisterDataSampleProcessor(
 
 	private final Map<String, List<DataSampleProcessor<DataSample>>>
 		_dataSampleProcessors = new ConcurrentHashMap<>();
+
+	@Reference(
+		target = "(destination.name=" + DestinationNames.MONITORING + ")"
+	)
+	private Destination _destination;
```
Comment (tinatian) [inline modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/messaging/MonitoringMessageListener.java:146]: @W2jx You do not need this dependency, please remove. Even if the destination is not ready, the listener is activated and registered, you won't be able to use the listener as there is no destination connected to it.

**Comment by tinatian** [comment]
@dantewang @W2jx Please see my comment and resend.

Tina

**Pull request closed without merging**